src/modules/SignIn/components/SignIn/components/FormSignIn/FormSignIn.jsx

```diff
@@ -100,18 +150,23 @@ export default class FormSignIn extends Component {
                         name="password"
                         placeholder="Password"
                         onChange={this.onFieldsChange}
-                        isValid={isValid.password}
                         value={formData.password}
+                        validationFunc={this.passwordValidor}
+                        ref={this.fieldRefs.password}
                     />
                     <InputGroup.Append>
                         <InputGroup.Text>
                             <FontAwesomeIcon icon={faLock} />
                         </InputGroup.Text>
                     </InputGroup.Append>
+                    <div className="error invalid-feedback">Password cannot be empty</div>
                 </InputGroup>
 
-                <div className="form-error-msg">{errorMsg}</div>
-                <Button variant="primary" type="submit" block>Sign in</Button>
+                <div className={`${classWarning} text-danger`}>
+                    {errorMsg}
```

Comment:
Let's not store the error message in the state and configs, but immediately put it in the div.